src/modules/user/index.ts

```diff
@@ -358,8 +358,54 @@ const getAvailableTimeZones = async (): Promise <any>  => {
   }
 }
 
+async function getEComStores(token: any, baseURL: string, vSize = 100, facilityId?: string): Promise<Response> {
```

Comment:
Rename it to something like `getEComStoresByFacility` as here we are fetching the products stores based on facility association and not all product Stores